### PR TITLE
Fix TRD geom transformation test

### PR DESCRIPTION
**Pull request by sawenzel**
Test was not actually testing anything since
TGeo geometry was not loaded.

### CODE REVIEW

File: Detectors/TRD/base/test/testCoordinateTransforms.cxx [hunk 1]
```diff
@@ -43,7 +44,13 @@ void testRCPoint(double calculatedPoint, double predictedPoint)
...
 {
+  // init ALICE geometry --> needed to instantiate TRD Geometry instance
+  auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbmgr.get<TGeoManager>("GLO/Config/GeometryAligned");
+
   auto mGeo = o2::trd::Geometry::instance();
+  BOOST_CHECK(mGeo);
+
   mGeo->createPadPlaneArray();
   mGeo->createClusterMatrixArray();
```
Comment (martenole) [inline Detectors/TRD/base/test/testCoordinateTransforms.cxx:55]: Actually, just this line here should be removed which causes the error to be printed. The cluster matrix array is not used in this test so this is not the issue

Comment (sawenzel) [reply Detectors/TRD/base/test/testCoordinateTransforms.cxx:55]: Yes, I agree. Will close this PR.

Comment (sawenzel) [reply Detectors/TRD/base/test/testCoordinateTransforms.cxx:55]: TRD should then remove the line `mGeo->createClusterMatrixArray`

Comment (bazinski) [reply Detectors/TRD/base/test/testCoordinateTransforms.cxx:55]: its removed in #11562

**Pull request closed without merging**